index.html

```diff
@@ -4,95 +4,99 @@
 		<link rel="stylesheet" type="text/css" href="style.css">
 		<title>Josh's Portfolio</title>
 	</head>
-		<body>
-			<header class="nav">
-				<a class="nav" href="#about">About Me</a>  
-				<a class="nav" href="#portfolio">Portfolio</a>
-				<a class="nav" href="#contact">Contact information</a>		
-			</header>
-			<section class="intro">
-				<p class="intro">HEELLL</p>
-				<p class="intro">OOOOO</p>
-				<p class="little-intro">I am Josh</p>
-				<p class="little-intro">Welcome to my world of</p>		
-				<p class="little-intro">organised chaos!</p>		
-			 </section>
+	<body>
+		<header class="id-section-bar">
+			<a class="id-section-bar" href="#about">About Me</a>  
+			<a class="id-section-bar" href="#portfolio">Portfolio</a>
+			<a class="id-section-bar" href="#contact">Contact information</a>		
+		</header>
```

Comment:
1. class name is a little confusing
2. parent and child should not have the same class name
3. in fact, you don't need a class here at all because your page should only have 1 header element anyway